plugins/filesystem/local/local.php

```diff
@@ -37,11 +37,26 @@ class PlgFileSystemLocal extends CMSPlugin
 	 */
 	public function onFileSystemGetAdapters()
 	{
-		// Compile the root path
+		$adapters = [];
+
+		// Compile the root path and load default images folder
 		$filePath = ComponentHelper::getParams('com_media')->get('file_path', 'images');
 		$root = JPATH_ROOT . '/' . $filePath;
 		$root = rtrim($root) . '/';
+		$adapters[] = new \Joomla\Plugin\Filesystem\Local\Adapter\LocalAdapter($root, $filePath);
```

Comment:
Add images as default value instead of hardcoding it, perhaps somebody doesn't want to have images.

Reply:
so shall we remove option from com_media ? we get images from there

Reply:
yes